src/endpoints/groups.js

```diff
@@ -12,6 +15,86 @@ const setGroup = (nameExtractor) => (req, res, next) => {
 };
 
 
+/* Group customizations
+ */
+
+
+/* Group overview
+ */
+
+
+/* GET
+ */
+const getGroupOverview = contentTypesProvided([
+  ["text/markdown", sendGroupOverview],
+  ["text/plain", sendGroupOverview],
+]);
+
+
+/* PUT
+ */
+const putGroupOverview = contentTypesConsumed([
+  ["text/markdown", receiveGroupOverview],
+]);
+
+
+/* DELETE
+ */
+const deleteGroupOverview = async (req, res) => {
+  authz.assertAuthorized(req.user, authz.actions.Write, req.context.group);
+
+  // TODO: somehow have a req.context.group.urlFor?
+  const groupSource = req.context.source;
+
+  const method = "DELETE";
+  const url = await groupSource.urlFor("group-overview.md", method);
```

Comment:
This doesn't work, I think because it's ES6 syntax but this project uses CommonJS.

This works though, noting that the name `method` is irrelevant to the interpreter (`foo="DELETE"` works just fine):

```js
urlFor("group-overview.md", method="DELETE");
```

Reply:
Oh nevermind, `method="DELETE"` fails on eslint's `no-undef`. My understanding is that `method="DELETE"` is an assignment that returns the value being assigned, based off of this [SO post](https://stackoverflow.com/q/16027247). It only "works" because the value is returned.

I don't think we want this here, so I'll just revert to the original code.

Reply:
So sorry, my suggestion misled you because I made a minor mistake! I didn't mean to suggest using named parameters. I thought the original call was `urlFor(…, {method})` and so suggested `urlFor(…, {method: "DELETE"})`. I should have suggested `urlFor(…, "DELETE")`.

Reply:
Ah ok, I see where you're coming from now. If the original call was `urlFor(…, {method})` then I would be fully on board with the original suggestion. However, I think I'll opt to keep it as-is, since:

1. Passing it directly as `urlFor(…, "DELETE")` loses the small bit of context that `"DELETE"` is an HTTP method. Even if that can be obvious with everything else around it, I prefer having the extra bit of context. Especially since my IDE (I assume others too) can't determine the function signature for `groupSource.urlFor`.
2. Consistency - `const method = "DELETE"` is also used elsewhere in the file.